.github/tlspuffin.matrix.json

```diff
@@ -259,9 +261,9 @@
       "preset": "wolfssl530-asan"
     },
     "cargo": {
-      "name": "wolfssl530",
+      "name": "",
```

Comment:
Please double check that the same CI tests get triggered (now with CPUT instead of Rust PUT); i.e., make sure the tests filtering nehave the same for CPUT and Rust PUT.